services/docker-compose.yml

```diff
@@ -245,16 +245,18 @@ services:
     volumes: *sidecar-volumes
     environment:
       <<: *sidecar-environment
-      DASK_SCHEDULER_ADDRESS: "tcp://dask-scheduler:8786"
-
+      DASK_SCHEDULER_ADDRESS: "tcp://${DASK_SCHEDULER_HOST:-dask-scheduler}:8786"
```

Comment:
what is the difference between DASK_SCHEDULER_HOST and DASK_SCHEDULER_ADDRESS  exactly??
could we have something more explicit? or is address implicitely linked to TCP protocol?

Reply:
DASK_SCHEDULER_ADDRESS  is the environ recognized by the [dask notebook](https://github.com/dask/dask-docker/blob/main/docker-compose.yml#L42), so I decided to keep the same .
Yes, it has to be tcp and I use the default port.
That said, i could also expose only DASK_SCHEDULER_HOST (as with the other services) and compose the variable inside